gcore/gdaldataset.cpp

```diff
@@ -7514,6 +7514,11 @@ void GDALDataset::ShareLockWithParentDataset(GDALDataset* poParentDataset)
     }
 }
 
+bool GDALDataset::SetQueryLoggerFunc(GDALQueryLoggerFunc /* callback */, void* /* context */ )
```

Comment:
doxygen comments to add.
You'll need to remove the /* and */, and add a CPL_UNUSED leading keyword (cf other similar cases in that file), to be able to document the parameters

Reply:
done